### PR TITLE
About section

### PR DESCRIPTION
**Pull request by KoffiKC**
About section created in line with Current Figma model. Also as part of the About section, a component for the app store buttons has been created, they currently only exist within the about section.

### CODE REVIEW

File: src/sections/About/About.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import UserProfile from '../../assets/images/UserProfile.png'
+import AppStoreButtons from '../../assets/AppStoreButtons/AppStoreButtons';
+
+const About = () => {
+    return (
+        <>
+            <section>
+                <div className='about-text'>
+                    <h1>Connecting those in Need</h1>
+                    <p>When one of us has food insecurity, it is a problem for all of us. Let’s change that, one meal at a time. <br /><br />
```
Comment (DrummerDee) [inline src/sections/About/About.jsx]: Remove break from this paragraph please :)

**Review by DrummerDee — CHANGES_REQUESTED** [review]

File: src/sections/About/About.jsx (hunk 1 above)
Comment (KoffiKC) [reply src/sections/About/About.jsx]: HI, have done :D

**Review by DrummerDee — APPROVED** [review]
Awesome sauce

**Comment by DrummerDee** [comment]
I like that you changed the break and fixed the positioning of the paragraph but it doesn't do too well with responsiveness. Maybe we could implement material ui and see if it could assist with responsiveness.

**Pull request merged**